test/utility/test_util.h

```diff
@@ -183,6 +201,18 @@ void *dmalloc(size_t);
 void *drealloc(void *, size_t);
 void *dstrdup(const void *);
 void *dstrndup(const char *, size_t);
+/*
+ * The op_ functions can generate errors that we wish to ignore. We have handler
```

Comment:
It's better not to reference parameter/variable names in comments. The `op_` above doesn't read nicely it'd be better replaced with `The following operation `